### DISCUSSION TITLE
Refactor Gopt

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/gopt#10] [General]
@tczorro should we merge the refactor branch into master and use that as the new base branch? 

I was looking at `Gopt` last Thursday with @FarnazH and it's clear we needed a refactor, but that was still the master branch; some of the issues are resolved in the refactor and I think that's what we'd want further development to be based on?

**Comment by tczorro** [comment]
@PaulWAyers It would be a good idea to use refactor as the new master. I think there are some undone work but it should not be the blocker. Those changes can be made later on. Or we might never get the refactor done
